<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Viewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: #ffffff;
    font-family: 'Segoe UI', sans-serif;
    color: #1f2937;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Image Information Viewer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 18pt;
    font-weight: 600;
    color: #111827;
    margin: 8px;
}</string>
     </property>
    </widget>
   </item>

   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Введите путь...</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 1px solid #d1d5db;
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 13px;
    background: #f9fafb;
}</string>
     </property>
    </widget>
   </item>

   <item row="2" column="0" rowspan="3">
    <widget class="QListView" name="listView">
     <property name="styleSheet">
      <string notr="true">QListView {
    border: 1px solid #d1d5db;
    border-radius: 6px;
    font-size: 13px;
    background: #ffffff;
    alternate-background-color: #f9fafb;
}</string>
     </property>
    </widget>
   </item>

   <item row="2" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image Info</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font-size: 13px;
    font-weight: 600;
    border: 1px solid #e5e7eb;
    border-radius: 6px;
    margin-top: 10px;
    padding-top: 6px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0"><widget class="QLabel" name="label_2"><property name="text"><string>File:</string></property></widget></item>
      <item row="0" column="1"><widget class="QLabel" name="FileName"><property name="text"><string>-</string></property></widget></item>
      <item row="1" column="0"><widget class="QLabel" name="label_4"><property name="text"><string>Size:</string></property></widget></item>
      <item row="1" column="1"><widget class="QLabel" name="Size"><property name="text"><string>-</string></property></widget></item>
      <item row="2" column="0"><widget class="QLabel" name="label_6"><property name="text"><string>Resolution:</string></property></widget></item>
      <item row="2" column="1"><widget class="QLabel" name="Resolution"><property name="text"><string>-</string></property></widget></item>
      <item row="3" column="0"><widget class="QLabel" name="label_8"><property name="text"><string>Color Depth:</string></property></widget></item>
      <item row="3" column="1"><widget class="QLabel" name="Color_depth"><property name="text"><string>-</string></property></widget></item>
      <item row="4" column="0"><widget class="QLabel" name="label_10"><property name="text"><string>Compression:</string></property></widget></item>
      <item row="4" column="1"><widget class="QLabel" name="Contraction"><property name="text"><string>-</string></property></widget></item>
     </layout>
    </widget>
   </item>

   <item row="3" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="multiChoice">
       <property name="text"><string>Select Multiple Images</string></property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: #3b82f6;
    color: white;
    border-radius: 6px;
    padding: 8px 16px;
    font-size: 13px;
    font-weight: 500;
}
QPushButton:hover { background: #2563eb; }
QPushButton:pressed { background: #1d4ed8; }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectFolderButton">
       <property name="text"><string>Select Folder</string></property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: #6b7280;
    color: white;
    border-radius: 6px;
    padding: 8px 16px;
    font-size: 13px;
    font-weight: 500;
}
QPushButton:hover { background: #4b5563; }
QPushButton:pressed { background: #374151; }</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Double-click on image file | Select multiple images or folder</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 11px;
    color: #6b7280;
    margin: 4px;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
